cogs/index_service_cog.py

```diff
@@ -95,8 +95,26 @@ async def discord_backup_command(self, ctx):
         await ctx.defer(ephemeral=True)
         await self.index_handler.backup_discord(ctx, user_api_key=user_api_key)
 
-    async def load_index_command(self, ctx, index):
+    async def load_index_command(self, ctx, user_index, server_index):
         """Command handler to backup the entire server"""
+
+        if not user_index and not server_index:
+            await ctx.respond("Please provide a user or server index")
+            return
+
+        if user_index and server_index:
+            await ctx.respond(
+                "Please provide only one user index or server index. Only one or the other."
+            )
+            return
+
+        if server_index:
```

Comment:
can u update readme slightly with the differences between user and server indexes